packages/calcite-components/src/components/tab-title/tab-title.scss

```diff
@@ -26,29 +26,29 @@
   .content {
     padding-inline-start: 32px; //28px button width + 0.25rem padding
   }
-}
 
-:host([layout="center"][bordered][closable]) .scale-s {
-  .content {
-    padding-inline-start: 36px; //28px button width + 0.5rem padding
+  .scale-l {
+    .content {
+      padding-inline-start: 40px; //36px button width + .25 padding
```

Comment:
Maybe to get rid of these comments we should have an internal var for button width and padding? Same goes for other uses below.